### PR TITLE
Add Pinned to Message struct

### PR DESCRIPTION
**Pull request by JurrijnP**

### CODE REVIEW

**Comment by JurrijnP** [comment]
Can somebody explain why this failed?

**Comment by Hinara** [comment]
You have a empty line with spaces which should not be there

File: message.go [hunk 1]
```diff
@@ -80,6 +80,9 @@ type Message struct {
 	// A list of reactions to the message.
 	Reactions []*MessageReactions `json:"reactions"`
 
+	// Whether the message is pinned or not.
+	Pinned bool `json:"pinned"`
+
```
Comment (Hinara) [inline message.go]: The empty line there should not have spaces

Comment (JurrijnP) [reply message.go]: Ah thanks will edit that

**Pull request merged**